sashimi-webapp/test/unit/specs/logic/processor/markdownProcessor.spec.js

```diff
@@ -0,0 +1,51 @@
+const myMod = require("src/logic/processor/markdownProcessor");
+
+describe('Test for MarkdownProcessor', () => {
+  it("should produce correct HTML format for Headers", () => {
+    const testH1 = myMod.process("# Hello World!");
+    const testH2 = myMod.process("## H2!");
+    const testH3 = myMod.process("### H3!");
+    const testH4 = myMod.process("#### H4!");
+    const testH5 = myMod.process("##### H5!");
+    const testH6 = myMod.process("###### H6!");
+    const testAltH1 = myMod.process("Alt-H1!\n======");
+    const testAltH2 = myMod.process("Alt-H2!\n------");
+
+    expect(testH1).to.equal("<h1>Hello World!</h1>\n");
```

Comment:
👍 

Initially we wanted to use `should` assertion style for our testing.  
However it seems like `expect` assertion style could be [better](http://softwareengineering.stackexchange.com/questions/255985/should-i-use-expect-or-should-when-writing-rspec-specs) for us. 

I guess we will stick to using `expect` syntax from now onwards.